internal/scaffold/cmd.go

```diff
@@ -127,11 +130,25 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
+	// Set default manager options
+	options := manager.Options{
 		Namespace:          namespace,
+		MapperProvider:     restmapper.NewDynamicRESTMapper,
```

Comment:
This RESTMapper is deprecated, and DynamicRESTMapper is used by [default](https://github.com/kubernetes-sigs/controller-runtime/blob/8fae388/pkg/manager/manager.go#L359) in the manager. Plus we shouldn't be making changes unrelated to the goals of the PR.

Reply:
good catcher 👍 tks